sunpy/net/helioviewer.py

```diff
@@ -88,11 +129,17 @@ def get_closest_image(self, date, **kwargs):
 
         return response
 
+    def display_source_id(self):
+        """
+        Returns a dictionary containing the source ID of all the 
+        images in the data source
+        """
+        return self._source_dict
```

Comment:
I question the need for this function. Can we not just access the dictionary directly instead?

Reply:
We can access the dictionary directly but we can keep that function if the user wishes to display the dictionary. It would be easy to use this function and print the dictionary

Reply:
But I don't see why we need a function to do this now that have a attribute that has all the information.

Reply:
I agree with nabobalis - we don't need this.  Users will use observatory, instrument, etc, and the source ID information is on the Helioviewer docs website.

Reply:
But this function can be used to directly display all the data sources that are available. In the rst file we can then directly use this function and print all the data sources. Won't that be easy for the user to use

Reply:
But you don't need a function that just returns the `._source_dict`. A user can access it directly (once we rename it)

Reply:
Yeah, tho it probably needs a better name.

Reply:
Ok, I will do that